src/pages/WorkflowSource.js

```diff
@@ -1,17 +1,17 @@
 import { Fragment } from 'react'
 import { div, h, h3 } from 'react-hyperscript-helpers'
-import { ButtonPrimary, IdContainer, makeInlineDockLink } from 'src/components/common'
+import { IdContainer, makeInlineDockLink } from 'src/components/common'
 import { TextArea } from 'src/components/input'
 import { FormLabel } from 'src/libs/form'
 
 
-export const WorkflowSource = ({ workflowUrl, setWorkflowUrl, onClick }) => {
+export const WorkflowSource = ({ workflowUrl, setWorkflowUrl }) => {
   return div([
     h3(['Provide a URL to a workflow']),
     div([
       h(Fragment, [
         `You can browse for workflows in `,
-        makeInlineDockLink('https://firecloud.dsde-dev.broadinstitute.org/?return=terra#methods', 'Broad Methods Repository', 14),
+        makeInlineDockLink('https://firecloud.dsde-dev.broadinstitute.org/?return=#methods', 'Broad Methods Repository', 14),
```

Comment:
I actually think we don't really want a return link at all? Because presumably it will return us to... a "singleton Cromwell" Terra page that's totally unrelated to the CaaA pages we linked from?

Reply:
So for some reason, without `return=#methods`, the URL redirects to Terra dev instead of loading Methods UI. Same thing for `?return=`. Hence I left it like this.